pages/index.js

```diff
@@ -26,6 +28,36 @@ const Home = (props) => {
     });
   }, []);
 
+  const fetchMembers = async () => {
+    try {
+      // assume API call returns array of members, but for now i'll use the arr of events
+      const response = await request.get("events/")
```

Comment:
For now, let's not use API calls as we are going to temporarily decouple the backend. All API calls have been removed in the latest master. It's fine to use constant values for the stats.